liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -277,3 +277,8 @@ info.undeploy.patternset.useraction=No action is required.
 info.undeploy.all=CWWKM2169I: Undeploying all applications.
 info.undeploy.all.explanation=All applications in the server will be undeployed.
 info.undeploy.all.useraction=No action is required.
+
+info.server.stop.noexist=CWWKM2170I: Server {0} does not exist.

```

Comment:
I updated code (#2) rather than touching the existing translated text. i.e. CWWKM2152I